apps/server/src/modules/user/repo/role.repo.integration.spec.ts

```diff
@@ -0,0 +1,227 @@
+import { MikroORM, NotFoundError } from '@mikro-orm/core';
+import { EntityManager, ObjectId } from '@mikro-orm/mongodb';
+import { MikroOrmModule } from '@mikro-orm/nestjs';
+import { Test, TestingModule } from '@nestjs/testing';
+import { MongoMemoryServer } from 'mongodb-memory-server';
+import { Role } from '../entity';
+import { RoleRepo } from './role.repo';
+
+describe('role repo', () => {
+	let module: TestingModule;
+	let mongodb: MongoMemoryServer;
+	let repo: RoleRepo;
+	let orm: MikroORM;
+	let em: EntityManager;
+
+	beforeAll(async () => {
+		mongodb = new MongoMemoryServer();
```

Comment:
use MongoMemoryDatabase Module instead, it just needs entities.then, in afterall, only module.close() will be required